app/views/navigation_rules/edit.html.erb

```diff
@@ -20,6 +20,12 @@
 
 <%= render 'shared/steps/form_notice', message: notice, status: "success" %>
 
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-6">
+    <%= render_markdown t("sidebar_settings.guidance_markdown") %>
+  </div>
+</div>
```

Comment:
Not sure if I should comment to you on this as the implementer but this copy feels a bit out of context when you browse this page.

![Screen Shot 2019-10-14 at 15 52 09](https://user-images.githubusercontent.com/282717/66760790-b2951600-ee9a-11e9-822a-41cb09abb2bd.png)

It feels like this is missing something that introduces the options - or maybe the explanations should be after the table? I get though that we're in a bit of a pickle as there isn't a spot for contextual guidance.

If it's all product ok'd happy to tick it though as code looks good.

Reply:
Agree with you that it feels a bit out of the blue, can raise it with product to see if they can enhance it.